src/shared/clients/s3Client.ts

```diff
@@ -287,6 +287,7 @@ export class DefaultS3Client {
             Key: request.key,
             Body: request.body,
             Expires: time,
+            ContentType: 'application/octet-stream',
```

Comment:
Are you sure this is wanted for the "GET" case? 
Should also call this out in the docstring.

Reply:
Updated to only set for the upload case.